admin_manual/ai/app_context_agent.rst

```diff
@@ -0,0 +1,59 @@
+==================================
+App: Context Agent (context_agent)
+==================================
+
+.. _ai-app-context_agent:
+
+The *context_agent* app is the apps that provide AI agent functionality in Nextcloud and acts as a backend for the :ref:`Nextcloud Assistant app<ai-app-assistant>`. Nextcloud can provide customer support upon request, please talk to your account manager for the possibilities.
+
+Requirements
+------------
+
+* This app is built as an External App and thus depends on AppAPI v3.1.0 or higher
+* Nextcloud AIO is supported
+* No GPU is necessary
+
+* CPU Sizing
+
+   * At least 1GB of system RAM
+
+Installation
+------------
+
+0. Make sure the :ref:`Nextcloud Assistant app<ai-app-assistant>` is installed
+1. :ref:`Install AppAPI and setup a Deploy Demon<ai-app_api>`
+2. Install the "Context Agent" ExApp via the "Apps" page in the Nextcloud web admin user interface
+3. Install a text generation backend like :ref:`llm2 <ai-app-llm2>` or :ref:`integration_openai <ai-ai_as_a_service>` via the "Apps" page in Nextcloud
+
+
+Model requirements
+~~~~~~~~~~~~~~~~~~
+
+This app requires underlying Large language models to support tool calling. The default model in *llm2* does *not* support tool calling. Instead we recommend:
+
+* Qwen 2.5 8B or higher
+* Watt Tool 8B or higher
```

Comment:
maybe we can switch the default model in llm2 in favour of one of these models.
nothing again this PR though.